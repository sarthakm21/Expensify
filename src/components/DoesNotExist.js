import React from "react";
const DoesNotExist = () => <div>404! The page requested does not exist.</div>;
export default DoesNotExist;
