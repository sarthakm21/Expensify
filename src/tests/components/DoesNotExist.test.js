import React from "react";
import DoesNotExist from "../../components/DoesNotExist";
import { shallow } from "enzyme";

test("should render doesNotExist component", () => {
  const wrapper = shallow(<DoesNotExist />);
  expect(wrapper).toMatchSnapshot();
});
