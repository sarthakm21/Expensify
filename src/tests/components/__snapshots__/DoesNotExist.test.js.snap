// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render doesNotExist component 1`] = `
<div>
  404! The page requested does not exist.
</div>
`;
